flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/JsonExistsOnError.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.expressions;
```

Comment:
move to API package?

Reply:
I placed it here since `BaseExpressions` already uses other enums which are placed here. I figured consistency is a better choice than breaking here.

Reply:
Maybe that was not well thought through. I'm fine with placing it there if both Java and Scala users do not have to add additional imports for it. But this is not the case. Not sure what is best.

Reply:
I think putting it under `api` is the best approach. Otherwise notebook implementations need to add a star import for `expressions` package.